Marlin/src/module/temperature.cpp

```diff
@@ -1203,11 +1199,7 @@ void Temperature::min_temp_error(const heater_id_t heater_id) {
  *  - Update the heated bed PID output value
  */
 void Temperature::manage_heater() {
-
-  #if EARLY_WATCHDOG
-    // If thermal manager is still not running, make sure to at least reset the watchdog!
-    if (!inited) return watchdog_refresh();
-  #endif
+  if (marlin_state == MF_INITIALIZING) return watchdog_refresh(); // If Marlin isn't started, at least reset the watchdog!
```

Comment:
We need this at all?

If we are initializing, we can call watchdog refresh once in setup. Or just init watchdog after all setup. If initialization takes more than watchdog timeout, we probably have a bigger problem.

Anyway, I'm ok with this version!

Reply:
The `setup()` is allowed to take as long as it takes, including users setting a very long boot-screen pause. For testing try setting a 5 second bootscreen timeout, and as always use dev mode to observe the `setup()` logging. If the watchdog resets, it will be clear where in the `setup` it happened.